wails-frontend/src/pages/success.tsx

```diff
@@ -1,14 +1,30 @@
 import { h } from 'preact';
 import { useLocation } from 'react-router';
+import { promptAction, promptRequest } from '../events/events';
 
 const Success = () => {
   const { state } = useLocation();
-  const { msg } = state;
+  const { req } = state;
+
+  const successMsg = (req: promptRequest) => {
+    switch (req.Action) {
+      case promptAction.deleteReq:
+        return 'The account has been deleted';
+      case promptAction.newPasswordReq:
+        return 'The password has been created';
+      case promptAction.importReq:
+        return 'The account has been imported';
+      case promptAction.signReq:
+        return 'The transaction has been signed';
+      default:
+        return 'Apply';
+    }
```

Comment:
maybe we should start setting up i18n for internationalization and it will simplify all the switch statements like this one?

Reply:
i don't understand how setup internationalization will simplify this :/

Reply:
because instead of the switch you use the translation key. i18n will do the switch for you

t(`prompt.action.${req.Action}`)

Reply:
> i don't understand how setup internationalization will simplify this :/

Neither I! Let's sync and you give more context. In any case is already merged. :)